a10_octavia/controller/worker/tasks/service_group_tasks.py

```diff
@@ -51,6 +56,11 @@ class PoolCreate(PoolParent, task.Task):
     @axapi_client_decorator
     def execute(self, pool, vthunder):
         try:
+            if pool.protocol == constants.PROTOCOL_PROXY:
+                raise exceptions.ProviderUnsupportedOptionError(
```

Comment:
Please include some information about the pool and vthunder. Ideally their ids. Operators will be using automation tools to build things so tracking this error down will be difficult.

Reply:
added pool id in log message. I cant accept it as a special variable, so embeded in same